File: Latihan B/java/Processor.java
// deklarasi kelas procssor
public class Processor{
    // variabel privat
    private String name;
    private int price;
    // method public dan konstruktor
    public Processor(){

    }
    public Processor(String name, int price){
        this.name=name;
        this.price=price;
    }
    // get dans set semua variabel
    public String getname(){
        return name;
    }
    public void setname(String name){
        this.name = name;
    }
    public int getprice(){
        return price;
    }
    public void setprice(int price){
        this.price = price;
    }
}

File: Latihan B/python/Processor.py
# deklarasi kelas
class Processor:
    # deklarasi variabel
    __name = ""
    __price = 0
    # method dan konstruktor
    def __init__(self, name = "", price = 0):
        self.__name = name
        self.__price = price
    #  get dans set semua variabel
    def setprice(self, price):
        self.__price = price
    def getprice(self):
        return self.__price
    def setname(self, name):
        self.__name = name
    def getname(self):
        return self.__name